.readthedocs.yml

```diff
@@ -10,10 +10,17 @@ build:
     os: ubuntu-22.04
     tools:
         python: '3.11'
-    commands:
-        - asdf plugin add uv
-        - asdf install uv 0.1.44
-        - asdf global uv 0.1.44
-        - uv venv
-        - uv pip install .[docs]
-        - .venv/bin/python -m sphinx -W --keep-going -d _build/doctrees -D language=en -b html docs/source $READTHEDOCS_OUTPUT/html
+    jobs:
+        pre_create_environment:
+            - asdf plugin add uv
+            - asdf install uv 0.5.17
+            - asdf global uv 0.5.17
+        create_environment:
+            - uv venv $READTHEDOCS_VIRTUALENV_PATH
+        install:
```

Comment:
Unlike with `uv sync` which we use in aiida-core, here we can manually specify the target virtual environment. This in turn let's us use the `sphinx` section below instead of specifying the build command manually.